src/backend/egl/context.rs

```diff
@@ -221,39 +221,50 @@ impl EGLContext {
         })
     }
 
+    /// Makes the OpenGL context the current context in the current thread with no surface bound.
+    ///
+    /// # Safety
+    ///
+    /// This function is marked unsafe, because the context cannot be made current on another thread without
+    /// being unbound again (see [`EGLContext::unbind`]).
+    pub unsafe fn make_current(&self) -> Result<(), MakeCurrentError> {
```

Comment:
Do we need a way to make the context not current if we expose this?

Reply:
`EGLContext::unbind` exists. (and is mentioned in the docs for exactly this function)

This also was already public btw.